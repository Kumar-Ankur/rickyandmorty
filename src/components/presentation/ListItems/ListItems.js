import React from "react";
import ListItem from "../ListItem/ListItem";

const ListItems = (props) => {
  const { filteredItem } = props;

  return filteredItem.map((item, indx) => {
    return <ListItem key={indx} item={item} indx={indx} />;
  });
};

export default ListItems;
