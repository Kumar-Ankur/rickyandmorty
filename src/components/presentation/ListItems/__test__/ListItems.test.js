import React from 'react';
import { shallow } from 'enzyme';
import ListItems from '../ListItems';

describe('check list items', () => {
    test('generate snapshot for listitems', () => {
        const wrapper = shallow(<ListItems filteredItem={[1,2,3]} />);
        expect(wrapper).toMatchSnapshot();
    })
})