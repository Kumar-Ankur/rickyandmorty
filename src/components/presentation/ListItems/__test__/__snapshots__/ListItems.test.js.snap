// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check list items generate snapshot for listitems 1`] = `ShallowWrapper {}`;
